src/main/webapp/app/shared/components/feature-overlay/feature-overlay.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, input } from '@angular/core';
+import { NgbTooltip } from '@ng-bootstrap/ng-bootstrap';
+import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
+
+@Component({
+    selector: 'jhi-feature-overlay',
+    template: `
+        <div ngbTooltip="{{ enabled() ? null : ('artemisApp.featureToggles.title' | artemisTranslate) }}" placement="left">
+            <div [class.disabled]="!enabled()">
+                <ng-content></ng-content>
+            </div>
+        </div>
+    `,
+    styles: [
+        `
+            .disabled {
```

Comment:
rather than custom style it would be nice to use Bootstrap. `pe-none` and `opacity-50` should do the same if I'm not mistaken.